test/runtests.jl

```diff
@@ -1,19 +1,20 @@
 using AWSBatch
-using AWSBatch: LogEvent, describe_jobs, describe_job_definitions, register_job_definition,
-    submit_job
-using AWSCore: AWSConfig, AWSException
-using AWSSDK.CloudWatchLogs: get_log_events
-using AWSTools.CloudFormation: stack_output
-using AWSTools.EC2: instance_region
+using AWS
```

Comment:
GH suggestions aren't allowing me to, but swapping lines 1&2 here to make things alphabetical.